### DISCUSSION TITLE
Is really possible to do a install withouth using CP machine ?

### DISCUSSION THREAD

**Discussion by agorosti** [HCL-TECH-SOFTWARE/connections-automation#206] [General]
Hi,
I tryed to use the ansible script to do a quickstart connections complete installation WITHOUT Connections pack..
The install finished ok, but I did realize that although CP machine is not going to be used, it needes to be started and provisioned.  In fact, the summary of the task of ansible, after the playbook finish,  have actions related to the CP machine...  see below example:

PLAY RECAP *************************************************************************************************************
connections.internal.example.com : ok=443  changed=204  unreachable=0    failed=0    skipped=227  rescued=0    ignored=17
cp.internal.example.com        : ok=6    changed=1    unreachable=0    failed=0    skipped=1    rescued=0    ignored=0

So, my question is ...    Why there is 6 actions done at the cp machine if is not going to be used (I do not need connections pack) ?  Why I do need to startup that machine?

Thks

**Comment by stoeps13** [comment]
If you don't want to install componentpack why do you define a componentpack server in inventory? There are tasks running on all servers when defined in inventory, just remove the cp.internal.example.com from all inventory groups. That should be enough.

**Reply by agorosti** [reply]
Thks. !
